.ci/dockerfiles/Dockerfile.gpu-test

```
@@ -23,12 +23,15 @@ ARG HAS_GPU=true
 ARG WORKSPACE=/workspace/nixl
 
 
+RUN sudo apt-get update && \
```

Hadolint (2.14.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)